### PR TITLE
add clarity of role use for eks clusterissuer

### PR DESCRIPTION
**Pull request by johndietz**
the only route53 dn01 clusterissuer example prior to this change specifies a role in each solver example, however in a single-account eks setup where the role is instead specified on the serviceaccount, including the same role on the clusterissuer causes sts errors and prevents the dns management.
i welcome any feedback on the comments i've provided in this eks clusterissuer example.
see discussion in https://github.com/jetstack/cert-manager/issues/3079 for more context on this issue.

### CODE REVIEW

**Comment by johndietz** [comment]
the entirety of this PR request is dco Signed-off-by: John Dietz <jdietz@virtru.com>
i tried rebasing the fork with a --signoff but the dco bot is still mad about the request.

File: content/en/docs/configuration/acme/dns01/route53.md [hunk 1]
```diff
@@ -233,4 +233,27 @@ serviceAccount:
     eks.amazonaws.com/role-arn: arn:aws:iam::XXXXXXXXXXX:role/cert-manager
 ```
 
+## Creating an Issuer (or `ClusterIssuer`)
```
Comment (meyskens) [inline content/en/docs/configuration/acme/dns01/route53.md:236]: It is me or am I missing some part you described in the PR here?

**Pull request closed without merging**